vsintegration/src/vs/FsPkgs/FSharp.LanguageService/FSharp.LanguageService.Base/CodeWindowManager.cs

```diff
@@ -14,7 +14,6 @@
 using Microsoft.VisualStudio.FSharp.LanguageService.Resources;
 
 namespace Microsoft.VisualStudio.FSharp.LanguageService {
-    /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager"]/*' />
```

Comment:
I have sent a question to our MSDN doc team about whether this will break, internal doc generation tooling.  My suspicion is that these comments are necessary.  But I honestly don't know.

In some ways that's the problem with all of this clean up, it is certainly needed but there are more teams that depend on, and can make changes to this codebase than just us.  That's not to say it isn't worth finding out, however, inadvertently breaking or even worse silently braking a bunch of teams who don't expect there to be any changes is always unfortunate.

I realize this seems silly, but I worry about these cleanup type PR's much more than the bug fix ones.  It is true I may have been at Microsoft too long ... because often the principle of least change is the one that guides our work.